AppAuth-JS/built/app/index.d.ts

```diff
@@ -0,0 +1,46 @@
+import { AuthorizationServiceConfiguration } from '../authorization_service_configuration';
```

Comment:
This is generated source file. All the files in built/ folder are generated JS file for the Google's lib fork